### ISSUE TITLE
Fix incorrect display of hotkeys

### ISSUE THREAD

**Issue by x9136** [Chatterino/chatterino2#4951]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

As you can see on the screenshot, to zoom in you need to press `⌘`+`+`, but it doesn't work for me. In fact, you need to press `Shift`+`⌘`+`+`. Perhaps it is related to the keyboard, because in order to write `+`, I need to press `Shift`.

I think that the list of hotkeys should show the real combination, so I suggest to take it into account if possible.

### Screenshots

![Screenshot 2023-11-06 at 14 47 21](https://github.com/Chatterino/chatterino2/assets/60105060/7f6b9890-a2e6-4bc6-97a9-3ab74626661b)

### OS and Chatterino Version

Chatterino 2.4.5 (commit 05a72d33) built with Qt 6.5.0 Running on macOS 14.1, kernel: 23.1.0

**Comment by Mm2PL** [comment]
This is a Qt issue. We do not stringify the keybindings ourselves.

```cpp
QString Hotkey::toString() const
{
    return this->keySequence().toString(QKeySequence::NativeText);
}
```

**Comment by x9136** [comment]
> Maybe try switching back to Qt 5 if that's an option?

Same thing.

Chatterino 2.4.0 (commit [a0f93100](https://github.com/Chatterino/chatterino2/commit/a0f93100) modified) built with Qt 5.15.2
Running on macOS 14.1, kernel: 23.1.0

**Comment by pajlada** [comment]
This is just how hotkeys are shown according to Qt

For key bindings like Command+Shift+N:
![image](https://github.com/Chatterino/chatterino2/assets/962989/a6e8ef13-0cf4-498f-870f-0377b42b3c65)
Command+Shift+Tab:
![image](https://github.com/Chatterino/chatterino2/assets/962989/29d74590-02df-4417-b5c5-c6a77dac90af)
Command+Plus:
![image](https://github.com/Chatterino/chatterino2/assets/962989/251251e0-295f-41b3-852b-1e802857ce73)

I'm assuming this is for the reason you mentioned in the original post

**Comment by x9136** [comment]
@pajlada Wrong keyboard shortcut in help is not a problem because that's how Qt works? Did I understand you correctly?

Or is it still a problem because the help gives false information? And it's better to write to Qt developers about it, because you have no way to fix it?

**Comment by pajlada** [comment]
@x9136 how do you press +?

**Comment by x9136** [comment]
@pajlada `Shift`+`+`


I did a little searching. This bug has been around for years and they can't quite get it fixed. If you're interested:
https://bugreports.qt.io/browse/QTBUG-51848

If I understand correctly, some developers are fixing it themselves. From the comments I learned this, for example:
![Screenshot 2023-11-08 at 14 19 48](https://github.com/Chatterino/chatterino2/assets/60105060/50eeb41b-5b5d-4b3c-9c7e-e70aed1e9c0f)

Anyway, I did the best I could on this one. Let me know if you do this someday and need to test some builds.
